src/commands/keyset.rs

```diff
@@ -1100,6 +1175,128 @@ impl Keyset {
     }
 }
 
+fn get_command(cmd: GetCommands, ksc: &KeySetConfig, kss: &KeySetState) {
+    match cmd {
+        GetCommands::UseCsk => {
+            println!("{}", ksc.use_csk);
+        }
+        GetCommands::Autoremove => {
+            println!("{}", ksc.autoremove);
+        }
+        GetCommands::KskAlgorithm => {
+            println!("{}", ksc.ksk_generate_params);
+        }
+        GetCommands::ZskAlgorithm => {
+            println!("{}", ksc.zsk_generate_params);
+        }
+        GetCommands::CskAlgorithm => {
+            println!("{}", ksc.csk_generate_params);
+        }
+        GetCommands::DsAlgorithm => {
+            println!("{}", ksc.ds_algorithm);
+        }
+        GetCommands::DnskeyLifetime => {
+            let span = Span::try_from(ksc.dnskey_signature_lifetime).expect("should not fail");
+            let signeddur = span
+                .to_duration(SpanRelativeTo::days_are_24_hours())
+                .expect("should not fail");
+            println!("{signeddur:#}");
+        }
+        GetCommands::CdsLifetime => {
+            let span = Span::try_from(ksc.cds_signature_lifetime).expect("should not fail");
+            let signeddur = span
+                .to_duration(SpanRelativeTo::days_are_24_hours())
+                .expect("should not fail");
+            println!("{signeddur:#}");
+        }
+        GetCommands::Dnskey => {
+            for r in &kss.dnskey_rrset {
+                println!("{r}");
+            }
+        }
+        GetCommands::Cds => {
+            for r in &kss.cds_rrset {
+                println!("{r}");
+            }
+        }
+        GetCommands::Ds => {
+            for r in &kss.ds_rrset {
+                println!("{r}");
+            }
+        }
+    }
+}
+
+fn set_command(
+    cmd: SetCommands,
+    ksc: &mut KeySetConfig,
+    config_changed: &mut bool,
+) -> Result<(), Error> {
+    match cmd {
+        SetCommands::UseCsk { boolean } => {
+            ksc.use_csk = boolean;
+            *config_changed = true;
```

Comment:
Since every match block sets config_changed = true, maybe move that out to the end?